tutorials/transfer_learning/dataloader.jl

```diff
@@ -16,7 +16,7 @@ const CATS = filter(x -> occursin("cat", x), FILES)
 # Takes in the number of requested images per batch ("n") and image size
 # Returns a 4D array with images and an array of labels
 function load_batch(n = 10, nsize = (224,224); path = PATH)
-  if ((batchsize % 2) != 0)
+  if ((n % 2) != 0)
```

Comment:
Perhaps clearer to use the named function?
```suggestion
  if isodd(n)
```